### ISSUE TITLE
Implement exit functionality for guided assistant progress

### ISSUE THREAD

**Issue by aroubin** [LSCOIT/LegalNavigator#351]
_From @yfzwinnie on September 6, 2018 18:14_

If user closes the broswer/tab during the guided assistant and they are not logged in, we should show a modal asking if they want to save their progress before exiting. If yes, ask user to login. If no, erase their progress.

_Copied from original issue: Microsoft/Access2Justice#448_

**Comment by aroubin** [comment]
This may no longer be needed, need to re-evaluate.

**Comment by svasylenko** [comment]
related (more detailed) task #350

**Comment by KellyeDC-zz** [comment]
Ready for PO review

**Comment by grawdon** [comment]
@KellyeDC When I either closed the browser tab or went to another website, nothing happened. It didn't looked resolved to me.

**Comment by KellyeDC-zz** [comment]
@svasylenko  Currently, if a platform end user is in the middle of an interview in the guided assist, is he/she able to save their place in the interview and comback to it, if registered?

**Comment by svasylenko** [comment]
@KellyeDC nope, user could bookmark a page but next time interview will start sfom beginning.

**Comment by KellyeDC-zz** [comment]
@svasylenko Did we implement this feature with #350  and #721 ?

**Comment by KellyeDC-zz** [comment]
If the user is registered, the idea is for the system to save the answer choices as the user progresses through the guided interview.  Has this been implemented?

**Comment by svasylenko** [comment]
@KellyeDC yes, please go trough interview, close browser at some point and open it back. Then login and go to GA - you will start from the question when you have closed the browser.
